src/components/FormAccordion.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div>
     <button
-      @click="showCollapse = !showCollapse"
+      v-b-toggle="`collapse-${config.label}`"
```

Comment:
Using `v-model`/`showCollapse` didn't allow the accordions to work in tandem so I had to use the `v-b-toggle` directive.